app/controllers/bookmarks_controller.rb

```diff
@@ -132,23 +132,22 @@ def status_action documents
     status = params['action']
     Array(documents.map(&:id)).each do |id|
       media_object = MediaObject.find(id)
-      if cannot? :update, media_object
-        errors += ["#{media_object.title} (#{id}) #{t('blacklight.messages.permission_denied')}."]
+      if status == 'publish'
+        success_ids << id if can?(:update, media_object)
```

Comment:
So with this current logic above, the user wouldn't get any error flash saying they cant update these. They'd just get shown nothing. Only way they get an error flash saying they dont have permission is if the mediaobject isn't in a published/unpublished state